docs/UserGuide.adoc

```diff
@@ -314,7 +325,7 @@ Input fields:
 
 Example:
 
-* `addAdmin user/master pass/myPass#1 n/John Doe sp/userdata.xml salary/4000 date/01/01/2018` +
+* `addAdmin user/myUsername pass/Password#1 n/John Doe sp/exampleconfig salary/3000 employ/30/09/2018` +
```

Comment:
`sp/exampleconfig` it would be good to include the constraint of the extension (e.g. `sp/examplecondfig.xml`) to stay consistent with the `save` command.

Reply:
Alright, updated to sp/exmapleconfig.xml